src/components/Calculator.js

```diff
@@ -0,0 +1,37 @@
+/* const calcHtmlElem = parser.parseFromString(calcStr, 'text/html').body.firstChild;
+calcWrapper.append(calcHtmlElem); */
+
```

Comment:
- [ ] **Commented codes should be removed from the codebase in order to keep it clean and therefore easy to read** 😃 👍 This is among the coding best practices. So, please removes these lines of code